packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -83,7 +82,7 @@ export class TSClient implements Generatable {
       generator,
       relativeEnvPaths,
       sqliteDatasourceOverrides,
-      relativePath: path.relative(outputDir, schemaDir),
+      relativePath: path.relative(outputDir, path.dirname(schemaPath)),
```

Comment:
I have ditched the `schemaDir` as it had little value, using `path.dirname(schemaPath)` is fine I believe.